### ISSUE TITLE
False positive with strict-component-boundaries and fixtures

### ISSUE THREAD

**Issue by alexandcote** [Shopify/eslint-plugin-shopify#116]
We have GraphQL fixtures in our project and we include these fixtures like that :
```import basicFixture from './fixtures/TestAdjacentResourcesQuery/basic.json';```

Because `TestAdjacentResourcesQuery` folder is PascalCase, the rule thinks it is a component but is not. We use PascalCase for that because the GraphQL file related to this fixture look like `TestAdjacentResourcesQuery.graphql`

Maybe we should exclude `fixtures` folder.

**Comment by beefchimi** [comment]
cc/ @cartogram

**Comment by TzviPM** [comment]
in quilt, we use fixture components in our tests. These are also causing a false positive:
<img width="737" alt="screen shot 2018-08-02 at 1 43 53 pm" src="https://user-images.githubusercontent.com/1950680/43600998-20fcadd2-965a-11e8-87c7-30108d839d76.png">

**Comment by GoodForOneFare** [comment]
Can the rule just accept an `excludes: glob[]` config option?
